pages/common.py

```diff
@@ -6,23 +6,97 @@
 import re
 import settings
 
-# Python 2 compatibility
-def nop2(x,y):
-    return x
-try:
-    unicode = unicode
-except:
-    unicode = nop2
-
-# This function will pass unicode text right through, other text will be decoded as utf-8
-def force_unicode(text):
-    if repr(text)[0] == 'u': #FIXME...
-        return text
-    return unicode(text.strip(),"utf-8")
+site_brand = "Freenet"
+
+# Wrapper for HTML intended to catch markdown-of-html bugs
+# The HTML class taints all strings that are added to it to be HTML as well
+class HTML(object):
+    def __init__(self, text):
+        if not isinstance(text, str):
+            raise TypeError("HTML text must be of str, {f} found".format(f=text.__class__.__name__))
+        self.text = text
+    
+    def __add__(self, other):
+        if isinstance(other, HTML):
+            return HTML(self.text + other.text)
+        return HTML(self.text + other)
+    
+    def __radd__(self, other):
+        if isinstance(other, HTML):
+            return HTML(other.text + self.text)
+        return HTML(other + self.text)
+    
+    def __repr__(self):
+        return u"HTML: " + self.text
```

Comment:
This was just an arbitrary choice. I'll dive into it.

As for the leading `u`: that's indeed an old Python 2.x habit. I'll make sure to drop it.